contribs/gmf/src/directives/displayquerygrid.js

```diff
@@ -86,7 +86,7 @@ gmf.displayquerygridComponent = {
   bindings: {
     'active': '=?gmfDisplayquerygridActive',
     'featuresStyleFn': '&gmfDisplayquerygridFeaturesstyle',
-    'selectedFeatureStyleFn': '&gmfDisplayquerygridSourceselectedfeaturestyle',
+    'selectedFeatureStyleFn': '&gmfDisplayquerygridSelectedfeaturestyle',
```

Comment:
So, now the attribute name will be: 
`gmf-displayquerygrid-selectedfeaturestyle`
(it was `gmf-displayquerygrid-sourceselectedfeaturestyle`)

It is what you want ?

Reply:
According to the doc, yes.

Reply:
And I don't see any use of it...